debian/changelog

```diff
@@ -1,14 +1,229 @@
-cloud-init (22.2-0ubuntu1~18.04.2) UNRELEASED; urgency=medium
+cloud-init (22.2-145-gaa59209a-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
 
   * d/control: add python3-debconf to Depends and Build-Depends
   * d/cloud-init.postinst: lintian fixes:
     + Fix command-with-path-in-maintainer-script for grub-install
-  * d/patches/renderer-do-not-prefer-netplan refresh to activators change
+  * d/p/renderer-do-not-prefer-netplan refresh to activators change
+  * d/p/expire-on-hashed-users.patch:
+    Add patch to ensure password expire doesn't apply to hashed users
   * d/source/lintian-overrides: lintian fixes:
     + silence binary-nmu-debian-revision-in-source bug:
       https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1014584
-
- -- Alberto Contreras <alberto.contreras@canonical.com>  Fri, 08 Jul 2022 12:13:38 +0200
+  * refresh patches:
+    + debian/patches/ec2-dont-apply-full-imds-network-config.patch
+    + debian/patches/openstack-no-network-config.patch
+    + debian/patches/renderer-do-not-prefer-netplan.patch
+  * New upstream snapshot. (LP: #SRU_BUG_NUMBER_HERE)
+    + Fix Chrony usage on Centos Stream (#1648) (LP: #1885952)
+      [Sven Haardiek]
+    + sources/azure: handle network unreachable errors for savable PPS
```

Comment:
This line adds a new lintian warning in sbuild.

Reply:
File a bug against lintian. Savable is a proper spelling. :wink: 

I'm wondering if our tooling will just regenerate this when it comes time for the actual release. We'll upstream snapshot then too. Will that regenerate this incorrect spelling? If so, I think it'd be better to wait.

@blackboxsw , do you know?

Reply:
> File a bug against lintian

Fwiw I tried and gave up because of learning curve with their platform :/

Maybe someone else will someday.

Reply:
Heh, that was actually a joke, but thanks for trying!

Reply:
> that was actually a joke, but thanks for trying!

I figured :)